Automattic/wp-calypso
# Add monetization tools features to complete lightbox

## Proposed Changes

* Add monetization tools feature copy to complete features

## Why are these changes being made?

Complete includes these features, and they used to be called "Creator", but now with Creator going away we need to list them specifically

## Testing Instructions

1. Open the Calypso Blue live link
2. Go to `/checkout/jetpack/jetpack_complete`
3. Ensure you see a line about the monetization tools
![Screenshot 2024-12-17 at 11 19 04 AM](https://github.com/user-attachments/assets/bd9d2fee-59e6-4fdb-9f83-55534978535c)
4. Open the Calypso Green live link
5. Go to `/pricing#jetpack_complete`
6. Ensure the monetization features are in the `Also Included` section
![Screenshot 2024-12-17 at 11 20 12 AM](https://github.com/user-attachments/assets/54e109c9-c9ed-49cf-98bb-a46bee357671)


## Pre-merge Checklist
